src/app/(tabs)/wheel-of-life.tsx

```diff
@@ -0,0 +1,43 @@
+import { Sector } from '@app/components';
+import { ANGLES, DATA } from '@app/constants';
+import React from 'react';
+import { StyleSheet, View, Dimensions } from 'react-native';
+import { G, Svg } from 'react-native-svg';
+
+const { width } = Dimensions.get('window');
+const size = width * 0.8;
```

Comment:
```suggestion
const SIZE = width * 0.8;
```
and the same for the rest of the constants